### PR TITLE
Merge recording branch into master

### PR DESCRIPTION
**Pull request by C2H6O**
We no longer run the "part-one" (master) part of this interview problem, let's merge the recording (part-two) into master to avoid confusion

### CODE REVIEW

**Review by bertodesk — APPROVED** [review]

File: app/src/main/java/com/plangrid/plansensor/MainActivity.kt [hunk 1]
```diff
@@ -14,6 +34,43 @@ class MainActivity : AppCompatActivity() {
...
+        val record = findViewById<Button>(R.id.record)
+        record.setOnClickListener {
+            // TODO: Implement recording
+        }
+        val play = findViewById<Button>(R.id.play)
+        play.setOnClickListener {
+            // TODO: Implement playback
+        }
+
+        // TODO(1): Listen to the sensor and display the last ten values in the graph.
```
Comment (alusch) [inline app/src/main/java/com/plangrid/plansensor/MainActivity.kt:46]: I think we also want to remove these `TODO(1)`s since they were confusing in the last interview we gave (having already been done in the part 2 branch). But that should be a separate PR

**Review by alusch — APPROVED** [review]
